lib/node_modules/@stdlib/blas/ext/base/ndarray/dlast-index-of/lib/main.js

```diff
@@ -78,14 +78,11 @@ function dlastIndexOf( arrays ) {
 	} else if ( fromIndex >= N ) {
```

Comment:
L76 also does not align with `Array.prototype.lastIndexOf`. Looks like you just copied these blocks from `index-of`, where those blocks are correct. Here, they are not correct. You need to fix the bounds checking behavior.